konduit-serving-core/src/main/java/ai/konduit/serving/launcher/command/ConfigCommand.java

```diff
@@ -0,0 +1,263 @@
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject pmml() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .modelConfig(
+                                        PmmlConfig.builder()
+                                                .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject dl4j() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .modelConfig(
+                                        DL4JConfig.builder()
+                                                .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private JsonObject keras() {
+        return new JsonObject(
+                new InferenceConfiguration(
+                        Arrays.asList(ModelStep.builder()
+                                .inputName("default")
+                                .outputName("default")
+                                .modelConfig(
+                                        KerasConfig.builder()
+                                                .build()
+                                )
+                                .build()),
+                        ServingConfig.builder()
+                                .build(),
+                        null
+                ).toJson()
+        );
+    }
+
+    private void printOrSave(String output) {
+        if(outputFile == null) {
+            log.info(output);
+        } else {
+            try {
+                FileUtils.writeStringToFile(outputFile, output, StandardCharsets.UTF_8);
+                log.info("Config file create successfully at {}", outputFile.getAbsolutePath());
+            } catch (IOException exception) {
+                log.error(String.format("Unable to save configuration file to %s", outputFile.getAbsolutePath()), exception);
```

Comment:
No String.format

Reply:
Sorry about that. Sometimes it doesn't work for me so I usually go with String.format. Maybe I have to explicitly set `{0}`, `{1}` inside the log string formats.

Reply:
I've never had to use anything other than `{}` in logging strings in all of DL4J and ND4J for logging - the only issue is that you don't need a {} for exceptions (it'll append that at the end with the stack trace)